static/code/editor.js

```diff
@@ -835,6 +835,8 @@ Code.functionEditor.createDom = function(container) {
     tabSize: 2,
     undoDepth: 1024,
     lineNumbers: true,
+    continueComments: 'Enter',
```

Comment:
1. Is this the same as `continueComments: true`, or is `Enter` not the same as the key that on my Mac is named "return"—i.e., enter?
1. If so, isn't this going to be kind of annoying?
1. If so, might we want to consider additionally specifying `continueLineComment: false`?

Reply:
Fixed:  `continueComments: {continueLineComment: false}`